sound/shairport-sync/patches/010-no-cxx.patch

```diff
@@ -1,7 +1,7 @@
 --- a/Makefile.am
```

Comment:
It seems like you are doing more changes than describing in the commit itself. You should add to the commit description that patch was refreshed.

Reply:
Got it. Updated.